rules/Transform/Rector/ClassMethod/WrapReturnRector.php

```diff
@@ -86,8 +86,9 @@ public function refactor(Node $node): ?Node
                     continue;
                 }
 
-                $this->wrap($classMethod, $typeMethodWrap->isArrayWrap());
-                $hasChanged = true;
+                if ($typeMethodWrap->isArrayWrap() && $this->wrap($classMethod)) {
```

Comment:
Instead of passing the isArrayWrap() value to the function, if it is not true we just don't call the function